soulstruct/darksouls1r/project/core.py

```diff
@@ -193,18 +193,11 @@ def offer_entities_export(self, with_window: ProjectWindow = None):
                 game_map = self.maps.GET_MAP(map_name)
 
                 if write_vanilla_entities_result == 0:
-                    try:
-                        vanilla_module = PACKAGE_PATH(
-                            f"darksouls1ptde/events/vanilla_entities/{game_map.emevd_file_stem}_entities.py"
-                        ).read_text()
-                    except FileNotFoundError:
-                        vanilla_module = ""
-                else:
-                    vanilla_module = ""
+                    self.include_vanila_entities = True
```

Comment:
Now the only logic needed here is setting the project configuration value.

Notably, this boolean can't be changed after project initialization (besides manually editing the config json, which will break a lot of things anyway), but I think this limitation really stems from that once an entity has replaced a hard-coded ID within a map script, there isn't a way to reverse this/update the entity without rolling back the script to use the ID again. Until that behavior changes, I think it's appropriate for this to be a one-time set.